autoload/slime.vim

```diff
@@ -62,18 +68,45 @@ function! s:SlimeRestoreCurPos()
   endif
 endfunction
 
+function! slime#send_range(startline, endline) abort
+  call s:TargetConfig()
```

Comment:
This causes crashes, I think this is not needed anymore anyway, since `slime#send` takes care of the configuration.

Reply:
pushed 3e745ae for this